### PR TITLE
added a dark theme

### PR DESCRIPTION
**Pull request by jpyllman**
This is the first step for supporting dark theme

### CODE REVIEW

**Comment by jpyllman** [comment]
So this is only a suggestion on how to add a dark theme. And the colors is not perfect.

File: popup/folderlist.js [hunk 1]
```diff
@@ -291,7 +296,7 @@ class TBFolderList extends HTMLElement {
     }
 
     this.selected = item;
-    this.shadowRoot.querySelector(".folder-list-body").focus();
+    //this.shadowRoot.querySelector(".folder-list-body").focus();
```
Comment (kewisch) [inline popup/folderlist.js:299]: Curious why you removed this? I think this was needed to make it possible to navigate with the keyboard.

File: popup/popup.css [hunk 2]
```diff
@@ -25,8 +43,20 @@ body, html {
...
 
 #folderList {
+  --folder-list-search-bgcolor: var(--menu-background-color-search);
+  --folder-list-search-color: var(--menu-color-search);
+
+  --folder-list-item-color: var(--menu-color);
+
   --folder-list-item-bgcolor-hover: var(--menu-background-color-select);
   --folder-list-item-color-hover: var(--menu-color-select);
+  /*--folder-list-item-border*/
```
Comment (kewisch) [inline popup/popup.css:53]: Did you want to make this one a rule too? If this is for future changes I'd prefer adding it in a different PR.

**Review by kewisch — CHANGES_REQUESTED** [review]
Can't believe its been almost a year. This looks almost ready, a few minor comments:

File: popup/folderlist.js (hunk 1 above)
Comment (jpyllman) [reply popup/folderlist.js:299]: As you see I put this in another PR. But I guess when I prepared this PR I already this comment in. If you want I can edit this PR.

File: popup/popup.css (hunk 2 above)
Comment (jpyllman) [reply popup/popup.css:53]: I do not remember why I put this in. I guess I copied it from some other example.

File: popup/folderlist.js (hunk 1 above)
Comment (kewisch) [reply popup/folderlist.js:299]: Yes, would appreciate if you could remove any commented out code from this PR. We can try to find a solution in your other PR for the focus issue.

Comment (kewisch) [inline popup/folderlist.js:299]:
```suggestion
    this.shadowRoot.querySelector(".folder-list-body").focus();
```

File: popup/popup.css (hunk 2 above)
Comment (kewisch) [reply popup/popup.css:53]:
```suggestion
```

**Review by kewisch — APPROVED** [review]

**Pull request merged**